tests/generator-Tests/Unit-Tests/CodeGeneratorExpectedResults/JavaInterop1/WriteKotlinUnsignedArrayTypeMethodsClass.txt

```diff
@@ -0,0 +1,90 @@
+// Metadata.xml XPath class reference: path="/api/package[@name='java.code']/class[@name='MyClass']"
```

Comment:
Why is this a `.txt` file?  It means we don't get C# highlighting on GitHub. :-(

Reply:
...because of SDK-style (short-form) `.csproj` project usage.  If these were `.cs` files, they'd be auto-included into the `.csproj` build, which is not what we want.